docs/zh/docs/network/modules/multus-underlay/dpdk.md

```diff
@@ -0,0 +1,251 @@
+# DPDK
+
+本篇文章主要是将在 DCE5.0 中如何快速的创建第一个 DPDK 应用.
+
+## 前置依赖
+
+- 安装 Multus-underlay , 并启用安装 SRIOV 组件, 参考[安装](install.md)。
+- 需要硬件支持: 拥有支持 SR-IOV 系列的网卡并设置 VFs,参考[SR-IOV](sriov.md)
+- 需要切换网卡驱动为用户态驱动:
+
+```shell
+# 下载 dpdk 源码
+root@master:~/cyclinder/sriov/# wget https://fast.dpdk.org/rel/dpdk-22.07.tar.xz && cd dpdk-22.07/usertools
+root@master:~/cyclinder/sriov/dpdk-22.07/usertools# ./dpdk-devbind.py --status
+root@172-17-8-120:~/cyclinder/sriov/dpdk-22.07/usertools# ./dpdk-devbind.py --status
+Network devices using kernel driver
+===================================
+0000:01:00.0 'I350 Gigabit Network Connection 1521' if=eno1 drv=igb unused=vfio-pci
+0000:01:00.1 'I350 Gigabit Network Connection 1521' if=eno2 drv=igb unused=vfio-pci
+0000:01:00.2 'I350 Gigabit Network Connection 1521' if=eno3 drv=igb unused=vfio-pci
+0000:01:00.3 'I350 Gigabit Network Connection 1521' if=eno4 drv=igb unused=vfio-pci
+0000:04:00.0 'MT27800 Family [ConnectX-5] 1017' if=enp4s0f0np0 drv=mlx5_core unused=vfio-pci *Active*
+0000:04:00.1 'MT27800 Family [ConnectX-5] 1017' if=enp4s0f1np1 drv=mlx5_core unused=vfio-pci *Active*
+0000:04:00.2 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v0 drv=mlx5_core unused=vfio-pci
+0000:04:00.3 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v1 drv=mlx5_core unused=vfio-pci
+0000:04:00.4 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v2 drv=mlx5_core unused=vfio-pci
+0000:04:00.5 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v3 drv=mlx5_core unused=vfio-pci
+0000:04:00.6 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v4 drv=mlx5_core unused=vfio-pci
+0000:04:00.7 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v5 drv=mlx5_core unused=vfio-pci
+0000:04:01.1 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v6 drv=mlx5_core unused=vfio-pci
+```
+
+以`0000:04:00.2 'MT27800 Family [ConnectX-5 Virtual Function] 1018' if=enp4s0f0v0 drv=mlx5_core unused=vfio-pci`: 为例：
+
+    a. 0000:04:00.2 : 该 VF PCI 地址
+    b. if=enp4s0f0v0 : 该 VF 网卡名称
+    c. drv=mlx5_core : 当前网卡驱动
+    d. unused=vfio-pci : 可切换的网卡驱动
+
+DPDK支持的用户态驱动有三种:
+    
+    a. vfio-pci: 在启用 IoMMU 情况下, 推荐使用此驱动, 性能安全性最好
+    b. igb-uio
```

Comment:
Is it possible to add some explanations like Line 42 and 44?

Reply:
Done.